### PR TITLE
Fix duplicate slot in exam schedule

### LINKED ISSUE #115: Slots get added additively on the exam schedule screen on acroll
## Bug Report

**Current Behavior**
The more we scroll random slots get added to the courses

**Steps to Reproduce**

1. Go to exam schedule 
2. Click on any exam
3. Scroll down to the last exam
4. Scroll back up to see random slots being added

**Expected Behavior**
Only slots of the courses must be shown

**Screenshots**

**Environment**

- Android version: Android 12
- Device make & model: Redmi Note 9 Pro Max 

**Possible Solution**
No idea

**Additional Context**
I think this occurs only after the latest update. It worked properly in CAT1

### PR DESCRIPTION
**Pull request by ezhil56x**
## Description
Duplicate slots are getting added when the user scrolls in the exam schedule. This has been fixed

Fixes #115

### CODE REVIEW

**Comment by therealsujitk** [comment]
Hi, thanks for contributing! Were you able to reproduce the linked issue? I don't think this would solve the issue, it might cause cards to have incorrect slots.

**Comment by ezhil56x** [comment]
I was able to reproduce the issue and as you said it caused the cards to have incorrect slots but now this works good. Can you review once?

File: app/src/main/java/tk/therealsuji/vtopchennai/adapters/ExamsItemAdapter.java [hunk 1]
```diff
@@ -88,6 +88,7 @@ public void setExamItem(Exam.AllData examItem) {
             TextView timings = this.examItem.findViewById(R.id.text_view_timings);
             TextView venue = this.examItem.findViewById(R.id.text_view_venue);
             ChipGroup slots = this.examItem.findViewById(R.id.chip_group_slots);
+            slots.removeAllViews();
```
Comment (therealsujitk) [inline app/src/main/java/tk/therealsuji/vtopchennai/adapters/ExamsItemAdapter.java:91]: Can you move this below to just above the for-loop?

Comment (ezhil56x) [reply app/src/main/java/tk/therealsuji/vtopchennai/adapters/ExamsItemAdapter.java]: Done!

**Comment by therealsujitk** [comment]
Awesome! Thanks!

**Review by therealsujitk — APPROVED** [review]

**Pull request merged**